src/cli.js

```diff
@@ -178,9 +178,14 @@ async function bundle(main, command) {
 
   const bundler = new Bundler(main, command);
 
-  if (command.name() === 'serve') {
+  console.log(command.target);
+
+  if (
+    (command.name() === 'serve' && command.target === undefined) ||
+    'browser'
```

Comment:
not sure I understand this logic

Reply:
that should've been `if (command.name() === 'serve' && (command.target === undefined ||  'browser')) {}` guess I made a typo